src/routes/Edit.svelte

```diff
@@ -14,14 +14,48 @@
     onMount(async () => {
         ga("send", "pageview");
         ga("send", "event", "version", mermaidVersion, mermaidVersion);
-        fromUrl(params.data);
+
+        const key = "historyList_key";
```

Comment:
Sorry for the misunderstanding, I meant the name of the variable(key). The value you had was perfectly fine.